keycloak/wait-for-keycloak.sh

```diff
@@ -2,5 +2,5 @@ DOT='\033[0;37m.\033[0m'
 ERROR='\033[0;31m[QUICKSTART] '
 RESET='\033[0m' # No Color
 
-while [[ "$(curl -s 'http://keycloak:9080/auth/realms/chorus' | jq '.realm')" != '"chorus"' ]]; do printf ${DOT}; sleep 5; done
+while [[ "$(curl -s 'http://keycloak:9080/realms/chorus' | jq '.realm')" != '"chorus"' ]]; do printf ${DOT}; sleep 5; done
```

Comment:
Nice change....

Reply:
We already use that in the Solr version of Chorus which was necessary to make the new Keycloak version work.